src/protocol/mod.rs

```diff
@@ -2110,6 +2110,1191 @@ impl<'input> Request<'input> {
         }
         Ok(Request::Unknown(header, remaining))
     }
+    // Get the matching reply parser (if any) for this request
+    pub fn reply_parser<'a>(&self) -> Option<ReplyParsingFunction<'a>> {
+        match self {
+            Request::Unknown(_, _) => None,
```

Comment:
Is this special case worth mentioning in the documentation? "Unknown requests are assumed not to have a reply"?

Reply:
Done.